cmd/gotk/utils.go

```diff
@@ -210,3 +217,81 @@ func (*Utils) makeDependsOn(deps []string) []dependency.CrossNamespaceDependency
 	}
 	return refs
 }
+
+func (*Utils) makeKustomizationYaml(dirPath string) error {
```

Comment:
Nitpick, but I feel like `build` or `generate` instead of `make` is a better description of what the method does.